packages/engine/src/scene/functions/loaders/ModelFunctions.ts

```diff
@@ -53,6 +55,9 @@ export const deserializeModel: ComponentDeserializeFunction = (
   addComponent(entity, ModelComponent, props)
 
   if (Engine.isEditor) getComponent(entity, EntityNodeComponent)?.components.push(SCENE_COMPONENT_MODEL)
+
+  initializeKTX2Loader(getGLTFLoader())
```

Comment:
i dont think we want to be initializing a new instance of the loader every time a model loads, this should be initialized once in the SceneObjectSystem initializer, same as the draco loader

Reply:
Added a null check to this such that it is only initialized once, and lazily as needed.